website/docusaurus.config.js

```diff
@@ -112,6 +112,11 @@ const config = {
                         label: "REST API Docs",
                         position: "left",
                     },
+                    {
+                        href: "https://ynput.github.io/ayon-python-api/build/html/ayon_api.html",
```

Comment:
this is actually a temporary path, my work is not done yet.
To be confirmed but final location will actually be `https://ynput.github.io/ayon-python-api/`